airbyte-integrations/connectors/source-hubspot/sample_files/configured_catalog.json

```diff
@@ -166,27 +166,19 @@
       "stream": {
         "name": "forms",
         "json_schema": {},
-        "supported_sync_modes": ["full_refresh"]
-      },
-      "sync_mode": "full_refresh",
-      "destination_sync_mode": "overwrite"
-    },
-    {
-      "stream": {
-        "name": "form_submissions",
```

Comment:
duplicate of https://github.com/airbytehq/airbyte/pull/11121/files#diff-451e5cb32beb551e49baaa86ea6d536087e1c40491d3ba71bb486c06a9da0c8bL185